code/controllers/subsystem/mapping.dm

```diff
@@ -51,16 +57,17 @@ SUBSYSTEM_DEF(mapping)
 	GLOB.space_manager.add_new_zlevel("Empty Area", linkage = CROSSLINKED, traits = list(REACHABLE))
 
 
-	// Setup the Z-level linkage
+	// Setup the Z-level linkage 
 	GLOB.space_manager.do_transition_setup()
 
 	// Spawn Lavaland ruins and rivers.
-	log_startup_progress("Populating lavaland...")
+	/* WEEBCITY - JTGSZ CHANGE - Disable lavaland ruin generation to save on init time/server resources unless we are using it.
+	log_startup_progress("Lavaland is disabled SEE: mapping.dm 62")
```

Comment:
This line won't be called as its inside a comment, so edit may not be that useful.

Reply:
thanks for the uhhh review, my bad on this one I wasn't paying much attention and it just picked the main repo

Reply:
Dont worry about it